### ISSUE TITLE
Question regarding evaluation of model

### ISSUE THREAD

**Issue by stodla123** [hudscomdz/scikit-anfis#1]
Hello,

unfortunatelly i wasn't able to fully understand how to evaluate a model given eg. in the tipping example:

How can i evalute the output for given inputs (linguistic variables) ?

Is there a way to just enter two variables (e.g. "Service" and "Food") and get and output ("Tip")

because this works:

input_tensor_eval=torch.tensor([[1, 2],
    ], dtype=dtype)
y_pred=model(input_tensor_eval)      
y1 = y_pred.detach().cpu()   
VariableOutput=float(y1[0])

but i want to enter in linguistic terms "good" and "bad" etc.

Thank you for your answer!
